includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -441,7 +441,7 @@ public function onArticleViewHeader( &$article, &$outputDone, &$useParserCache )
 		$title = $article->getTitle();
 
 		$namespace = $title->getNamespace();
-		if ( !empty( $this->namespaceRegistry[$namespace] ) &&  $title->exists() ) {
+		if ( !empty( $this->namespaceRegistry[$namespace] ) ) {
 			$this->wg->Out->addHTML(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)